### PR TITLE
ones_like API: remove device, input -> x

### PR DESCRIPTION
**Pull request by zhupengyang**
### PR types

Others

### PR changes

APIs

### Describe

- ones_like API
    - remove device attr
    - input -> x
    - update doc
image

### CODE REVIEW

File: python/paddle/tensor/creation.py [hunk 1]
```diff
@@ -164,74 +164,49 @@ def ones(shape, dtype=None, name=None):
...
+
     Returns:
-        out(Variable): The tensor variable storing the output.
+        Variable: A ones tensor with the same shape of ``x``. The data type of
+        the output is the same as ``x``, or if ``dtype`` is not None, it is the
+        same as ``dtype``.
+
+    Raise:
+        TypeError: If ``dtype`` is not None and does not point to bool,
+            float16, float32, float64, int32 or int64.
```
Comment (wangchaochaohu) [inline python/paddle/tensor/creation.py]: 这里改成not one of bool, float是不是好点

File: python/paddle/tensor/creation.py [hunk 2]
```diff
@@ -164,74 +164,49 @@ def ones(shape, dtype=None, name=None):
...
-def ones_like(input, dtype=None, device=None, name=None):
+def ones_like(x, dtype=None, name=None):
     """
 	:alias_main: paddle.ones_like
-	:alias: paddle.ones_like,paddle.tensor.ones_like,paddle.tensor.creation.ones_like
+	:alias: paddle.tensor.ones_like, paddle.tensor.creation.ones_like
 
-    This function creates a ones tensor which has identical shape and dtype 
-    with `input`.
+    This OP returns a ones tensor with the same shape of ``x``. The data type
```
Comment (wangchaochaohu) [inline python/paddle/tensor/creation.py]: a ones tensor 改成 a tensor which all value is 1?

File: python/paddle/tensor/creation.py (hunk 1 above)
Comment (zhupengyang) [reply python/paddle/tensor/creation.py]: Done

File: python/paddle/tensor/creation.py (hunk 2 above)
Comment (zhupengyang) [reply python/paddle/tensor/creation.py]: Done

File: python/paddle/tensor/creation.py [hunk 3]
```diff
@@ -164,74 +164,47 @@ def ones(shape, dtype=None, name=None):
...
-def ones_like(input, dtype=None, device=None, name=None):
+def ones_like(x, dtype=None, name=None):
     """
 	:alias_main: paddle.ones_like
-	:alias: paddle.ones_like,paddle.tensor.ones_like,paddle.tensor.creation.ones_like
+	:alias: paddle.tensor.ones_like, paddle.tensor.creation.ones_like
 
-    This function creates a ones tensor which has identical shape and dtype 
-    with `input`.
+    This OP returns a Tensor filled with the value 0, with the same shape and
```
Comment (lanxianghit) [inline python/paddle/tensor/creation.py]:
```suggestion
    This OP returns a Tensor filled with the value 1, with the same shape and
```

File: python/paddle/tensor/creation.py [hunk 4]
```diff
@@ -164,74 +164,47 @@ def ones(shape, dtype=None, name=None):
...
-            return out
-    helper.append_op(
-        type='fill_any_like',
-        inputs={'X': [input]},
-        attrs=attrs,
-        outputs={'Out': [out]})
-    out.stop_gradient = True
-    return out
+    """
+    return full_like(x=x, fill_value=1, dtype=dtype, name=name)
```
Comment (lanxianghit) [inline python/paddle/tensor/creation.py:207]: 报错信息有添加吗

File: python/paddle/tensor/creation.py (hunk 3 above)
Comment (zhupengyang) [reply python/paddle/tensor/creation.py]: Done

File: python/paddle/tensor/creation.py (hunk 4 above)
Comment (zhupengyang) [reply python/paddle/tensor/creation.py:207]: Done

**Review by lanxianghit — APPROVED** [review]
LGTM

**Review by swtkiwi — APPROVED** [review]
LGTM

**Pull request merged**